front/components/assistant/conversation/SidebarMenu.tsx

```diff
@@ -127,7 +127,7 @@ export function AssistantSidebarMenu({ owner }: AssistantSidebarMenuProps) {
       Yesterday: [],
       "Last Week": [],
       "Last Month": [],
-      "Last Year": [],
+      "Last 12 Months": [],
```

Comment:
Can we create a type for this? And properly type the `groups` constant? TS should definitely prevent such issue.

Reply:
Can we use a `as const` instead of creating a separate type (kind of painful to maintain, and takes space)

Reply:
@fontanierh I'm not sure what you mean, can you show me ?

Reply:
Actually we can't otherwise the arrays are readonly. NVM me separate type is the way to go !